openjdk8/src/test/resources/structure.yaml

```diff
@@ -0,0 +1,24 @@
+# Test spec for the Cloud Runtimes container structure testing framework.
+#
+# See https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/structure_tests/README.md for more detail.
+#
+schemaVersion: 1.0.0
+
+commandTests:
+- name: 'java version is 1.8'
+  command: ['java', '-version']
+  expectedError: ['openjdk version "1\.8.*"']
```

Comment:
We might be able to, though the `$openjdk8version` maven property and the output of `java -version` aren't quite in the same format. I'm not sure if there's a nice way of reliably comparing between the two. Ideally I'd like not to make this string comparison overly strict.

`$openjdk8version` = `8u111` 
`java -version` contains `openjdk version 1.8.0_111`